lib/kubeclient/inflections.rb

```diff
@@ -0,0 +1,4 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'resource_quota', 'resource_quotas'
+  inflect.irregular 'resourcequota', 'resourcequotas'
+end
```

Comment:
@tenderlove @matthewd it seems that the inflector doesn't pluralize the word quota.
I saw it also here
http://codereply.com/answer/8hnvfd/method-pluralize-seems-working.html
I wanted to make sure that the above solution is the recommended one.
(Also based on what's written here https://github.com/rails/rails/blob/5373bf228d1273deae0ed03370ec4a63c580422b/activesupport/lib/active_support/inflector/methods.rb#L11)